src/components/settings/SettingsLMSTab/index.jsx

```diff
@@ -62,17 +62,19 @@ const SettingsLMSTab = ({
     openLmsStepper();
   }, [dispatch, openLmsStepper]);
 
-  // we pass in params (configId and lmsType) from SyncHistory when user wants to edit that config
   useEffect(() => {
     const query = new URLSearchParams(window.location.search);
-    const fetchData = async () => channelMapping[query.get('lms')].fetch(query.get('id'));
-    fetchData()
-      .then((response) => {
-        editExistingConfig(camelCaseObject(response.data), query.get('id'));
-      })
-      .catch((err) => {
-        logError(err);
-      });
+    // if we have passed in params (configId and lmsType) from SyncHistory, user wants to edit that config
+    if (query.size > 0) {
```

Comment:
IMO we might as well check for the presence of `lms` and `id` in the query, just in case more query parameters come in later.